components/server/src/workspace/gitpod-server-impl.ts

```diff
@@ -1343,7 +1347,11 @@ export class GitpodServerImpl implements GitpodServerWithTracing, Disposable {
         const uniqueURLs = new Set();
         return suggestions
             .sort((a, b) => {
-                // Most recently used first
+                // priority first
+                if (a.priority != b.priority) {
+                    return a.priority > b.priority ? 1 : -1;
```

Comment:
Should this be sorted in descending order? (I.e. highest priority first?)

```suggestion
                    return a.priority < b.priority ? 1 : -1;
```